src/hotspot/cpu/x86/interp_masm_x86.cpp

```diff
@@ -399,7 +399,7 @@ void InterpreterMacroAssembler::check_and_handle_earlyret(Register java_thread)
 
     movptr(tmp, Address(rthread, JavaThread::jvmti_thread_state_offset()));
     testptr(tmp, tmp);
-    jcc(Assembler::zero, L); // if (thread->jvmti_thread_state() == NULL) exit;
+    jcc(Assembler::zero, L); // if (thread->jvmti_thread_state() == null) exit;
```

Comment:
nullptr

Reply:
Missed change to nullptr as you suggested.